conf/i18n/translations/messages.pot

```diff
@@ -360,14 +359,14 @@ msgctxt "Example: 1-10 of 50"
 msgid "<span class=\"yxt-ResultsHeader-resultsCountStart\">[[start]]</span> <span class=\"yxt-ResultsHeader-resultsCountDash\">-</span> <span class=\"yxt-ResultsHeader-resultsCountEnd\">[[end]]</span> <span class=\"yxt-ResultsHeader-resultsCountOf\">of</span> <span class=\"yxt-ResultsHeader-resultsCountTotal\">[[resultsCount]]</span>"
 msgstr ""
 
-#: src/ui/templates/results/verticalresultscount.hbs:5
+#: src/ui/templates/results/verticalresultscount.hbs:3
 msgctxt "Example: 1-10 of 50"
 msgid ""
 "<span class=\"yxt-VerticalResultsCount-start\">[[start]]</span>\n"
-"<span class=\"yxt-VerticalResultsCount-dash\">-</span>\n"
-"<span class=\"yxt-VerticalResultsCount-end\">[[end]]</span>\n"
-"<span class=\"yxt-VerticalResultsCount-of\">of</span>\n"
-"<span class=\"yxt-VerticalResultsCount-total\">[[resultsCount]]</span>"
+"    <span class=\"yxt-VerticalResultsCount-dash\">-</span>\n"
+"    <span class=\"yxt-VerticalResultsCount-end\">[[end]]</span>\n"
+"    <span class=\"yxt-VerticalResultsCount-of\">of</span>\n"
+"    <span class=\"yxt-VerticalResultsCount-total\">[[resultsCount]]</span>"
```

Comment:
Is it necessary to add a tab to these rows? Since that changed the msgid, that requires us to update all of the .po files to match it. I believe that's what caused the translation test to fail

Reply:
I just removed this extra spacing because it didn't look like it was required in order to fix the issue, and because it messed up the translation ids